rules/windows/process_creation/win_susp_tracker_execution.yml

```diff
@@ -0,0 +1,31 @@
+title: DLL Injection with Tracker.exe
+id: 148431ce-4b70-403d-8525-fcc2993f29ea
+description: This rule detects DLL injection and execution via LOLBAS - Tracker.exe
+author: Avneet Singh '@v3t0_', oscd.community
```

Comment:
could you please put the entire author field to the quotes? 
this way:

```
author: 'Avneet Singh @v3t0_, oscd.community'
```

Reply:
sure, I've modified the author field. And this is a faulty PR as it includes the commits from other branches as well. So I'll close this one and will issue a new PR.